src/providers/CPKWeb3Provider.js

```diff
@@ -122,13 +122,15 @@ class CPKWeb3Provider extends CPKProvider {
     return { hash };
   }
 
-  encodeMultiSendCalldata(multiSend, txs) {
+  static encodeMultiSendCallData({ web3, transactions, multiSendAddress }) {
+    const multiSend = new web3.eth.Contract(multiSendAbi, multiSendAddress);
```

Comment:
Instead of `multiSendAddress`, use a constant `'0x0000000000000000000000000000000000000000'`, as this doesn't actually need a deployed contract for encoding.